src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -87,6 +87,8 @@ data class IndividualUser(
     val firstName: String?,
     val lastName: String?,
     val countryCode: String?,
+    val cookiesConsented: Boolean?,
```

Comment:
Should this be a map? In most places I see these consent banners, the consent is either "all", "none", or some subset of a set of cookies. Perhaps that is not necessary yet though?

Reply:
The requirements doc specifically says it should be a boolean. Our setup is very simple: we only have essential cookies (no consent needed) and analytics ones, no other categories.